dlc_developer_config.toml

```diff
@@ -0,0 +1,28 @@
+[dev]
+# Set to "huggingface", for example, if you are a huggingface developer
+partner_developer = false
```

Comment:
is this a boolean value or string?

Reply:
What is the significance for this field?

Reply:
Good point, probably a more sensible default is ""

This is basically to signal that a build is associated with a given partner